deploy/modules/azure/tidb-cluster/main.tf

```diff
@@ -0,0 +1,87 @@
+# The name of a node pool may only contain lowercase alphanumeric characters and must begin with a lowercase letter.
```

Comment:
We do not maintain the tidb-cluster chart anymore, suggest using the TidbCluster CR directly.

Reply:
Good to know, thanks!